### ISSUE TITLE
ability to read a stream as well as file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Cornutum/tcases#1]
```
What steps will reproduce the problem?
None.

What is the expected output? What do you see instead?
ability to read a stream as well as file, passed to tcases.main method.

What version of the product are you using? On what operating system?
1.3.0

Please provide any additional information below.


```

Original issue reported on code.google.com by `kpgx...@gmail.com` on 29 Sep 2014 at 6:38

**Comment by kerrykimbrough** [comment]
Thanks, I see the issue now. A peek at the source for Tcases.run() offers a clue about the way to go. But I agree that this ought to be easier to. The solution is a bit involved since there are actually 3 inputs to the generation process, including the optional generator definition and base test case definitions. Look for a fix in the next release=1.3.1

**Comment by kerrykimbrough** [comment]
Fixed in 1.3.1

Tcases: To better support embedding in other apps, add stream-based methods for generating and exporting test case definitions.
